### ISSUE TITLE
Questions on cityscapes mask rcnn experiment reproduction and dataset

### ISSUE THREAD

**Issue by JiaxingGeng95** [facebookresearch/Detectron#734]
I am currently having a hard time to reproduce mask rcnn results on cityscapes. The AP I got is only around 0.3 with COCO pretrained model, and I can't figure out what's wrong with my training. I used config file of R50-FPN with changes mentioned in the mask rcnn paper. Is there other hyperparam that is critical but not mentioned in the paper?

I found that the val and training dataset seem to be prepared differently, where one is filtered and one is not. It seems that the conversion for filtered output is given in tools folder, but what does this "filter" really mean and should I prepare training dataset the same way?
https://github.com/facebookresearch/Detectron/blob/master/detectron/datasets/dataset_catalog.py#L51

Thanks!
